docs/API.md

```diff
@@ -144,8 +145,19 @@ gulp.task('build', ['array', 'of', 'task', 'names']);
 **Note:** The tasks will run in parallel (all at once), so don't assume that the tasks will start/finish in order.
 
 #### fn
+Type: `Function`
+
+The function that performs the task's main operations. Generally this takes the form of:
 
-The function that performs the task's operations. Generally this takes the form of `gulp.src().pipe(someplugin())`.
+```js
+gulp.task('buildStuff', function(someArguments) {
```

Comment:
no arguments are passed here other than callback and that's only if you specify the `cb` argument